ownCloudScreenshotsTests/ownCloudScreenshotsTests.swift

```diff
@@ -0,0 +1,177 @@
+//
+//  ownCloudScreenshotsTests.swift
+//  ownCloudScreenshotsTests
+//
+//  Created by Javier Gonzalez on 19/03/2019.
+//  Copyright © 2019 ownCloud GmbH. All rights reserved.
+//
+
+/*
+* Copyright (C) 2018, ownCloud GmbH.
+*
+* This code is covered by the GNU Public License Version 3.
+*
+* For distribution utilizing Apple mechanisms please see https://owncloud.org/contribute/iOS-license-exception/
+* You should have received a copy of this license along with this program. If not, see <http://www.gnu.org/licenses/gpl-3.0.en.html>.
+*
+*/
+
+import XCTest
+import EarlGrey
+import LocalAuthentication
+import ownCloudSDK
+
+class ScreenshotsTests: XCTestCase {
+
+	let url = "localhost"
+	let user = "admin"
+	let password = "admin"
+	let serverDescription = "ownCloud"
+
+    override func setUp() {
+		super.setUp()
+        continueAfterFailure = false
+    }
+
+    override func tearDown() {
+		super.tearDown()
+    }
+
+	func testTakeScreenshotStep() {
+
+		let app = XCUIApplication()
+		app.launchEnvironment = ["oc:app.show-beta-warning": "false", "oc:app.enable-ui-animations": "false"]
+		setupSnapshot(app)
+		app.launch()
+
+		snapshot("01_screenshot")
+
+		//Settings
+		app.toolbars["Toolbar"]/*@START_MENU_TOKEN@*/.buttons["settingsBarButtonItem"]/*[[".buttons[\"Settings\"]",".buttons[\"settingsBarButtonItem\"]"],[[[-1,1],[-1,0]]],[0]]@END_MENU_TOKEN@*/.tap()
```

Comment:
- Indention 
- Comments are a little bit confusing

Reply:
I added the comments just to identify each part of code that have involved on change the view.
Not sure what do you see there wrong